frontend/package.json

```diff
@@ -28,6 +29,7 @@
   "devDependencies": {
     "@types/bootstrap": "^5.2.5",
     "@types/node": "^16.11.56",
+    "@types/yup": "^0.32.0",
```

Comment:
Running the build locally, npm reports that @types/yup entry isn't needed.

Reply:
I have removed @types/yup entry